### PR TITLE
Implement vaccination certification card (EXPOSUREAPP-7438)

### PR DESCRIPTION
**Pull request by jurajkusnier**
device-2021-06-03-164732

### CODE REVIEW

**Review by mtwalli — APPROVED** [review]
Lgtm, Tested on S8

File: Corona-Warn-App/src/main/res/layout/vaccination_home_immune_card.xml [hunk 1]
```diff
@@ -5,73 +5,90 @@
...
-        app:layout_constraintTop_toBottomOf="@id/card_title" />
+        app:layout_constraintTop_toBottomOf="@id/title1" />
+
+    <ImageView
+        android:id="@+id/shield_icon"
+        android:layout_width="24dp"
+        android:layout_height="24dp"
+        android:layout_marginStart="@dimen/card_padding"
+        android:importantForAccessibility="no"
+        android:src="@drawable/ic_white_shield_with_checkmark"
```
[the remark this answers is not in the document]
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/res/layout/vaccination_home_immune_card.xml]: updated

File: Corona-Warn-App/src/main/res/layout/vaccination_home_card.xml [hunk 2]
```diff
@@ -5,92 +5,90 @@
...
-        app:layout_constraintTop_toBottomOf="@id/card_title" />
+        app:layout_constraintTop_toBottomOf="@id/title1" />
+
+    <ImageView
+        android:id="@+id/shield_icon"
+        android:layout_width="24dp"
+        android:layout_height="24dp"
+        android:layout_marginStart="@dimen/card_padding"
+        android:importantForAccessibility="no"
+        android:src="@drawable/ic_half_white_half_gray_shield"
```
[the remark this answers is not in the document]
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/res/layout/vaccination_home_card.xml]: updated

**Review by BMItr — APPROVED** [review]
tested on onePlus 8. Lgtm

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Pull request merged**